MVCDemos/Views/Autocomplete/Overview.cshtml

```diff
@@ -88,15 +89,53 @@
     var firstName = "",
         lastName = "",
         position = "CEO",
-        city = "",
-        state = "";
+        state = "",
+        currentClient = "";
+
+    var autoComplete_customStore = new DevExpress.data.CustomStore({
```

Comment:
Why did u name it as `autoComplete_customStore`? I think it's better to name it `clientsStore` as in another frameworks

Reply:
I followed the naming format in the https://demos.devexpress.com/ASPNetCore/Demo/TreeList/LoadDataOnDemand/ demo. I agree that it's better to use `clientStore` in this case so I updated it.